compiler/compiler.go

```diff
@@ -1519,6 +1519,8 @@ func (b *builder) createBuiltin(argTypes []types.Type, argValues []llvm.Value, c
 			case *types.Pointer:
 				ptrValue := b.CreatePtrToInt(value, b.uintptrType, "")
 				b.createRuntimeCall("printptr", []llvm.Value{ptrValue}, "")
+			case *types.Slice:
+				b.createRuntimeCall("printslice", []llvm.Value{value}, "")
```

Comment:
You need to split up the slice in pointer/len/cap and cast the pointer to a consistent type (like unsafe.Pointer).

Reply:
I had that originally but this works the same. Is there any reason to use one over the other?

Reply:
Yes, try printing a slice that's not a byte slice. []int for example.

Reply:
After I updated the tests with your suggestion I noticed this 😅

Reply:
Done.